packages/xo-web/src/common/xo/index.js

```diff
@@ -1010,29 +1032,55 @@ export const startVms = vms =>
       )
     )
 
-    if (forbiddenStart.length === 0) {
+    if (forbiddenStart.length === 0 && vmsWithduplicatedMacAddresses.length === 0) {
       if (nErrors === 0) {
         return
       }
 
       return error(_('failedVmsErrorTitle'), _('failedVmsErrorMessage', { nVms: nErrors }))
     }
 
-    const choice = await chooseActionToUnblockForbiddenStartVm({
-      body: _('blockedStartVmsModalMessage', { nVms: forbiddenStart.length }),
-      title: _('forceStartVmModalTitle'),
-    }).catch(noop)
-
-    if (nErrors !== 0) {
-      error(_('failedVmsErrorTitle'), _('failedVmsErrorMessage', { nVms: nErrors }))
+    if (vmsWithduplicatedMacAddresses.length > 0) {
+      // Retry without checking MAC addresses
+      try {
+        await confirm({
+          title: _('forceStartVm'),
+          body: _('vmsWithDuplicatedMacAddressesMessage', { nVms: vmsWithduplicatedMacAddresses.length }),
+        })
+        await Promise.all(
+          map(vmsWithduplicatedMacAddresses, id =>
+            _startVm(id, undefined, { bypassMacAddressesCheck: true }).catch(reason => {
+              if (forbiddenOperation.is(reason)) {
+                forbiddenStart.push(id)
+              } else {
+                nErrors++
+              }
+            })
+          )
+        )
+      } catch (error) {
+        // Nothing to do
+      }
     }
 
-    if (choice === 'clone') {
-      return Promise.all(map(forbiddenStart, async id => cloneAndStartVm(getObject(store.getState(), id))))
-    }
+    if (forbiddenStart.length > 0) {
+      // Clone or retry with force
+      const choice = await chooseActionToUnblockForbiddenStartVm({
+        body: _('blockedStartVmsModalMessage', { nVms: forbiddenStart.length }),
+        title: _('forceStartVmModalTitle'),
+      }).catch(noop)
+
+      if (nErrors !== 0) {
+        error(_('failedVmsErrorTitle'), _('failedVmsErrorMessage', { nVms: nErrors }))
+      }
```

Comment:
Do you know why there's an error shown here? IMO it should also count the failed retries below, and only then show an error with the amount of VMs that failed to restart.

Reply:
- I just kept the same behavior as before so as not to make too many changes.

- `nErrors`  will also contain the amount of VMs that we tried to restart, even if they contain duplicate MAC addresses

Reply:
- Ok
- That's not what I understand from the code: every time we call `VM.start`, either the VM is pushed into one of the retry queue collections *or* `nErrors` is incremented, not both.

Reply:
I will fix this in another PR